packages/yoroi-extension/app/components/wallet/send/WalletSendForm.js

```diff
@@ -484,23 +484,19 @@ export default class WalletSendForm extends Component<Props> {
             options={sendAmountOptions}
             {...form.$('selectedAmount').bind()}
             onChange={value => {
-              this.form.$('selectedAmount').value = value
-              if(value === CUSTOM_AMOUNT && this.props.shouldSendAll) {
-                this.props.toggleSendAll();
+              if (value === CUSTOM_AMOUNT) {
+                this.props.updateSendAllStatus(false);
+                this.form.$('amount').clear();
               } else if (value !== CUSTOM_AMOUNT) {
```

Comment:
@AhmedIbrahim336 , you can replace `else if (value !== CUSTOM_AMOUNT)` with just `else` here, because you have the checks like:

```
if (x === y) {

} else if (x !== y) {

}
```

So it's same as 

```
if (x === y) {

} else {

}
```